### DISCUSSION TITLE
Data sources: Recreational

### DISCUSSION THREAD

**Discussion by melissamonk-NOAA** [pfmc-assessments/CA_quillback_rockfish_2025#2] [General]
List of recreational data to compile and analyze

1. Catches
- [x] a. Ralston reconstruction (same the one used for 2021 assessment). For 1928-1980. ; not availalbe at a finer scale than N/S 
- [x] b. MRFSS catch data pulled October 17, 2024. For 1980-1989, 1993-2004
  - [x] See #25 for details on filling in missing years
- [x] c. RecFIN catch data pulled October 2, 2024. For 2005-2023
  - [x] Need to request new proxy values for 2020, 2021 (See [the california data repo](https://github.com/pfmc-assessments/california-data/blob/main/recreational-fishery/proxy%202020%20data/READ%20ME%20Rockfish%20Genus%20Allocation.txt) for details from the 2023 cycle) - Requested 11/24/2024. These are split into two types of adjustments:
    - [x] The 2020 COVID proxy values (Provided 12/4/2024 - pfmc-assessments/california-data#9)
    - [x] The 2020-2021 rockfish genus allocations  (pfmc-assessments/california-data#8)
2. Composition data
- [x] a. RecFIN length data pulled October 2, 2024. For 2004-2023
  - [x] Need to develop comps for the model
- [x] b. MRFSS length data pulled November 22, 2024. For 1980-1989, 1993-2003. 
  - [x] See #19 for details on duplication of MRFSS data for 1997-1998. Will replace PC records in this year with Deb Wilson-Vandenberg data (converted to fork length using conversions in #5) 
  - [x] Need to develop comps for the model
- [x] c. Deb-Wilson Vandenberg onboard observer (CPFV). For 1987-1998. 
  - [x] Need to develop comps for the model
- [x] d. Miller and Geibel and Miller and Gotshall (PC and PR). For 1959-1960
  - [x] Need to develop comps for the model
- [x] e. Geibel data from North shore (Del Norte and Humboldt) PR fleet. For 1992-1998  
  - [x] Need to develop comps for the model 
3. Age composition data
- [x] Right now we are splitting these for growth fleets. We have the following comps for non-commercial fleets
  - [x] a. Noncommercial all (growth fleet)
  - [x] b. Noncommercial CCFRP (possibly as an age comp - though this would be fishery independent)
  - [x] c. Noncommercial non-CCFRP (growth fleet)
  - [x] a-c but as CAAL 
5. Discards
- [x] a. Discards available within RecFIN.
  - [x] Need to confirm MRFSS 2004 data have discard estimates. UPDATE: They do but they are treated like other MRFSS data
6. Indices
- [x] a. Deb-Wilson Vandenberg onboard observer
- [x] b. Recent CPFV onboard observer (**UPDATE**: Requested from Ross and Paulo on 11/20/2024)
- [x] c. Private/rental dockside (**UPDATE**: Requested from Laura and Ashok on 11/20/2024)
- [x] d. Geibel northshore PR survey?
- [x] e. MRFSS charter and ~private~ boat indices

See Data sources: Biological for details on recreational data biology

**Comment by brianlangseth-NOAA** [comment]
Im looking over the recfin bio data. I noticed a few with a source of CPFV as opposed to BIO. While CPFV's could be sampled (and would be Private/Charter) I didn't think any direct CPFV sampling was within recfin. There are very few of these (33 out of nearly 6000), and all were released. Is it a different sampling process for CPFV compared to BIO?

**Comment by brianlangseth-NOAA** [comment]
The fields for MRFSS are still opaque. My rec code has a number of questions about field names that I could not find answers for from EJ's provided metadata doucment as linked in this disscussion pfmc-assessments/california-data#2

**Comment by brianlangseth-NOAA** [comment]
See above that we are missing 2004 for rec catches. Im unsure where to obtain 2004 rec catch data. The values for MRFSS are the same between the data I just processes and the data from the 2021 assessment. That file had 2004 values so could copy there. 

Also, note that the reconstructions stated that they believe their estimate for 1980 is more reliable than the MRFSS 1980 estimate so plan to apply historical value for 1980.

**Reply by brianlangseth-NOAA** [reply]
I emailed Jason Edwards about the lack of 2004 data for MRFSS and he confirmed that this is indeed an error. He fixed it on 10/17/2024 and CA MRFSS now goes through 2004. Resolved with d2c67ec

**Comment by brianlangseth-NOAA** [comment]
CRFS district map [link](https://wildlife.ca.gov/Conservation/Marine/CRFS/Survey-Design) and map below

![image](https://github.com/user-attachments/assets/ff4cd835-2862-4496-b164-8b7efc1fd8ef)

**Comment by brianlangseth-NOAA** [comment]
@JuliaCoates looked at MRFSS PR data and found only 84 quillback from 1995-2003, and thus percent positive is very low, and therefore this dataset will not be used for an index.

**Comment by melissamonk-NOAA** [comment]
The only recreational index for the Stock Synthesis model that has enough information is the CDFW PR dockside CRFS era index.
